src/presign/participant.rs

```diff
@@ -1145,8 +1145,14 @@ impl PresignKeyShareAndInfo {
             let r2_pub_j = round_three_input.r2_public.clone();
             let r2_priv_j = round_three_input.r2_private.clone();
 
-            let alpha = self.aux_info_private.sk.decrypt(&r2_pub_j.D)?;
-            let alpha_hat = self.aux_info_private.sk.decrypt(&r2_pub_j.D_hat)?;
+            let alpha = self
+                .aux_info_private
+                .decryption_key()
+                .decrypt(&r2_pub_j.D)?;
+            let alpha_hat = self
+                .aux_info_private
+                .decryption_key()
+                .decrypt(&r2_pub_j.D_hat)?;
```

Comment:
note: this just changes direct access to `sk` to controlled call to `decryption_key()`